modeling-cmds/src/websocket.rs

```diff
@@ -93,11 +93,25 @@ pub enum WebSocketRequest {
 }
 
 /// A sequence of modeling requests. If any request fails, following requests will not be tried.
-#[derive(Serialize, Deserialize, JsonSchema, Debug, Default)]
+#[derive(Serialize, Deserialize, JsonSchema, Debug)]
 #[serde(rename_all = "snake_case")]
 pub struct ModelingBatch {
     /// A sequence of modeling requests. If any request fails, following requests will not be tried.
     pub requests: Vec<ModelingCmdReq>,
+    /// ID of batch being submitted.
+    /// Each request has their own individual ModelingCmdId, but this is the
+    /// ID of the overall batch.
+    pub id: ModelingCmdId,
+}
```

Comment:
This will generate other rs code that will allow me to do WebSocketRequest::ModelingCmdBatchReq { requests, cmd_id } ?

Reply:
can confirm this will show up in kittycad.rs as ModelingCmdBatchReq